include/crm/common/output_internal.h

```diff
@@ -161,6 +161,17 @@ pcmk__output_t *pcmk__mk_xml_output(char **argv);
 #define PCMK__SUPPORTED_FORMAT_TEXT { "text", pcmk__mk_text_output, pcmk__text_output_entries }
 #define PCMK__SUPPORTED_FORMAT_XML  { "xml", pcmk__mk_xml_output, pcmk__xml_output_entries }
 
+/*!
+ * \brief Enable various tweaks to the XML formatted output
+ */
+enum pcmk__xml_subformat {
+    pcmk__xml_none,                 // Regular XML formatted output
+    pcmk__xml_legacy,               // Only for backwards compatability in crm_mon
```

Comment:
Let's add a COMPAT note for `pcmk__xml_legacy`. I hope we can remove `crm_mon --as-xml` at 3.0.0. I'm less confident about getting rid of `pcmk__xml_list_element`, since that requires more schema changes...